src/shared/Microsoft.Git.CredentialManager/Interop/MacOS/MacOSEnvironment.cs

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+using System;
+using System.IO;
+using Microsoft.Git.CredentialManager.Interop.Posix;
+
+namespace Microsoft.Git.CredentialManager.Interop.MacOS
+{
+    public class MacOSEnvironment : PosixEnvironment
+    {
+        public override bool IsSamePath(string a, string b)
+        {
+            a = Path.GetFullPath(a);
+            b = Path.GetFullPath(b);
+
+            // TODO: check if APFS/HFS+ is in case-sensitive mode
```

Comment:
First the first release can we leave the above as a TODO?

Reply:
Commit is from a previous PR (now merged). This is OK for the preview. Once a user hits a problem we can reevaluate. Most macOS users don't change this.